libs/icu/Makefile

```diff
@@ -41,23 +41,36 @@ define Package/icu
   DEPENDS:=+libstdcpp +libpthread
 endef
 
+CONFIGURE_ARGS_ICU := \
+	--disable-debug \
+	--enable-release \
+	--enable-shared \
+	--enable-static \
+	--enable-draft \
+	--enable-renaming \
+	--disable-tracing \
+	--disable-extras \
+	--enable-dyload \
+	--enable-layout \
+	--enable-layoutex \
+				
+CONFIGURE_ARGS += $(CONFIGURE_ARGS_ICU) \
+	--disable-tools \
+	--disable-tests \
+	--disable-samples \
+	--with-cross-build="$(HOST_BUILD_DIR)"
+	
+HOST_CONFIGURE_ARGS += $(CONFIGURE_ARGS_ICU)
+
 define Build/Configure
```

Comment:
I am looking at using this but I have been unable to get it to compile as the args do not seem to get passed. Not sure but `configure` looks hard coded in _host-build.mk_?
```
define Host/Configure/Default
	$(if $(HOST_CONFIGURE_PARALLEL),+)(cd $(HOST_BUILD_DIR)/$(3); \
		if [ -x configure ]; then \
			$(CP) $(SCRIPT_DIR)/config.{guess,sub} $(HOST_BUILD_DIR)/$(3)/ && \
			$(2) \
			$(HOST_CONFIGURE_CMD) \
			$(HOST_CONFIGURE_VARS) \
			$(HOST_CONFIGURE_ARGS) \
			$(1); \
		fi \
	)
endef
```